Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM golang:alpine
+MAINTAINER Dave Lasley <dave@laslabs.com>
+
+# Install Build Dependencies
+
+ENV buildDeps "build-base \
+               gcc \
+               git \
+               libtool \
+               sqlite-dev"
+
+RUN apk add --no-cache $buildDeps
+
+# Install CFSSL
+
+RUN git clone --depth=1 https://github.com/cloudflare/cfssl.git $GOPATH/src/github.com/cloudflare/cfssl
+
+WORKDIR $GOPATH/src/github.com/cloudflare/cfssl
+
+RUN set -x \
+	&& go get github.com/GeertJohan/go.rice/rice \
+    && rice embed-go -i=./cli/serve \
+	&& cp -R $GOPATH/src/github.com/cloudflare/cfssl/vendor/github.com/cloudflare/cfssl_trust /etc/cfssl \
+	&& go build -o /usr/bin/cfssl ./cmd/cfssl \
+	&& go build -o /usr/bin/cfssljson ./cmd/cfssljson \
+	&& go build -o /usr/bin/mkbundle ./cmd/mkbundle \
+	&& go build -o /usr/bin/multirootca ./cmd/multirootca \
+	&& apk del $buildDeps \
+	&& rm -rf $GOPATH/src \
+	&& echo "Build complete."
+
+# Create and Change to PKI Dir
+RUN mkdir -p /var/pki
```

Comment:
`-p` is not necessarily required because `/var` will always exist but it also isn't hurting anything :)